TKGame/Enemy/KnightEnemy.cs

```diff
@@ -1,44 +1,20 @@
-﻿using System;
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
 using System.Collections.Generic;
-using System.Drawing;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using Microsoft.Xna.Framework;
-using Microsoft.Xna.Framework.Graphics;
 
-
-namespace TKGame.Enemy
+namespace TKGame
 {
-    class DoomguyEnemy : Enemy
+    public class KnightEnemy : Enemy
     {
-        static DoomguyEnemy instance;
-        private static object syncRoot = new object();
-
-        public static DoomguyEnemy Instance
+        public KnightEnemy() 
         {
-            get
-            {
-                if (instance == null)
-                {
-                    lock (syncRoot)
-                    {
-                        if (instance == null)
-                        {
-                            instance = new DoomguyEnemy();
-                        }
-                    }
-                }
-                return instance;
-            }
-        }
-
-        private DoomguyEnemy()
-        {
-            entityTexture = Art.EnemyTexture;
-            Position = new Vector2(150, 700);
+            entityTexture = Art.KnightEnemyTexture; 
+            Position = new Vector2(300, 800); // hard coded spawn position at the moment
             velocity = new Vector2((float)1.5, 1);
```

Comment:
We should initialize this to 0 and have an input/physics/graphics component for entity transformation